public/assets/css/header.css

```diff
@@ -0,0 +1,27 @@
+body > header > nav > ul > li:first-child {
+    width: 64px;
+    height: 62px;
```

Comment:
why width : 64px and height: 62px ?
i think width and height should be in the same size (width: 64px height: 64px)